example/install_rackhd.sh

```diff
@@ -0,0 +1,252 @@
+#!/bin/bash
+#
```

Comment:
Missing set -e and error handling throughout the script.  Can you add the checks to validate that the commands completed as you expected them to?  ie: check that the apt-get install worked, the wgets worked, etc.